analysis/pfe_methods/combined_patch_subset_method_test.py

```diff
@@ -87,9 +87,9 @@ def test_differs_by_network(self):
     desktop_session.page_view(usage)
     twog_session.page_view(usage)
 
-    self.assertNotEqual(
-        desktop_session.get_request_graphs()[0].total_response_bytes(),
-        twog_session.get_request_graphs()[0].total_response_bytes())
+    # self.assertNotEqual(
```

Comment:
This assert should still work since it's testing desktop vs 2g which should have different params